pkg/platform/model/vcs.go

```diff
@@ -600,34 +601,18 @@ func CommitInitial(hostPlatform string, langName, langVersion string) (strfmt.UU
 	return res.Payload.CommitID, nil
 }
 
-func isWildcardVersion(version string) bool {
-	return strings.Index(version, "x") >= 0 || strings.Index(version, "X") >= 0
-}
-
-func VersionStringToConstraints(version string) ([]*mono_models.Constraint, error) {
-	if !isWildcardVersion(version) {
-		return []*mono_models.Constraint{{Comparator: "eq", Version: version}}, nil // exact version
+func versionStringToConstraints(version string) ([]*mono_models.Constraint, error) {
```

Comment:
Not asking for a change, but I just realized I was wrong in the first review in that we don't use the mono model. Filed a followup here: https://activestatef.atlassian.net/browse/DX-2064